### PR TITLE
chore(schema.prisma): Update Representations table schema

### PR DESCRIPTION
**Pull request by Philip-Philippou1**
Representations schema to change based on Service User changes

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by Yakitrak — COMMENTED** [review]
James knows much more about this so will leave the approval to him but had a question - why can't we do both migrations in one go so instead of committing two folders in one PR, we can do it in one?

File: packages/applications-service-api/prisma/migrations/20231129122831_remove_rep_redacted_fd/migration.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  - You are about to drop the column `redacted` on the `Representation` table. All the data in the column will be lost.
+
+*/
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[Representation] DROP COLUMN [redacted];
```
Comment (JamesP-KC) [inline packages/applications-service-api/prisma/migrations/20231129122831_remove_rep_redacted_fd/migration.sql:12]: You could combine the two migrations into one. The idea being to take this `ALTER TABLE` line and put it after the `DROP CONSTRAINT` in the other migration

File: packages/applications-service-api/prisma/schema.prisma [hunk 2]
```diff
@@ -61,40 +61,13 @@ model Representation {
   referenceId                    String?
   status                         String?
   dateReceived                   DateTime?
-  originalRepresentation         String    @db.NVarChar(4000)
-  redacted                       Boolean   @default(false)
-  redactedRepresentation         String?   @db.NVarChar(4000)
-  redactedBy                     String?
-  redactedNotes                  String?   @db.NVarChar(4000)
+  representationComment          String?   @db.NText
```
Comment (JamesP-KC) [inline packages/applications-service-api/prisma/schema.prisma:64]: `NText` is deprecated which is why it was `NVarChar(4000)` before. Back Office are using `NText` so we can go for consistency (and both have to switch when it is fully deprecated), or use `NVarChar` which is the [recommended replacement](https://learn.microsoft.com/en-us/sql/t-sql/data-types/ntext-text-and-image-transact-sql?view=sql-server-ver16) and preempt it

File: packages/applications-service-api/prisma/migrations/20231129122831_remove_rep_redacted_fd/migration.sql (hunk 1 above)
Comment (Philip-Philippou1) [reply packages/applications-service-api/prisma/migrations/20231129122831_remove_rep_redacted_fd/migration.sql]: Good shout and much better merging into 1.

File: packages/applications-service-api/prisma/schema.prisma (hunk 2 above)
Comment (Philip-Philippou1) [reply packages/applications-service-api/prisma/schema.prisma:64]: Originally my concern was that BO could have a 4100 char rep and we would truncate.  Thought I'd make it consistent now then we can align on length in future.

**Review by JamesP-KC — APPROVED** [review]

**Pull request merged**